src/docs/PML/PMLUserGuide.adoc

```diff
@@ -1572,21 +1572,27 @@ There are numerous research papers and studies dedicated to the analysis of the
 
 To access the AutoFeat page, please follow the steps below:
 
-Open the link:https://try.activeeon.com/automation-dashboard/#/portal/workflow-execution[Workflow Execution Portal].
+Open the link:https://try.activeeon.com/studio[Studio Portal].
```

Comment:
Why do you ask to open in it Studio. It should be WE

Reply:
Actually, all dataset-specific variables are defined within task variables such as the FILE_PATH, delimiter, etc.,  Otherwise, having them as workflow variables (as before) can be blocking in some use cases using two different datasets in the same ML workflow. As a consequence, those variables cannot be edited/specified from the WE.